src/components/RJST/index.tsx

```diff
@@ -204,21 +205,25 @@ export const RJST = <T extends RJSTBaseResource<T>>({
 	const internalOnChange = React.useCallback(
 		(
 			updatedFilters: JSONSchema[],
-			sortInfo: TableSortOptions | null,
+			sortInfo: TableSortOptions<T> | null,
 			page: number,
 			itemsPerPage: number,
 		) => {
 			if (!onChange) {
 				return;
 			}
+
 			const pineFilter = pagination?.serverSide
 				? convertToPineClientFilter([], updatedFilters)
 				: null;
 			const oData = pagination?.serverSide
 				? pickBy(
 						{
 							$filter: pineFilter,
-							$orderby: orderbyBuilder(sortInfo, customSort),
+							$orderby:
+								sortInfo && typeof sortInfo.sortable === 'string'
+									? sortInfo.sortable
+									: orderbyBuilder(sortInfo, customSort),
```

Comment:
NIT: How about instead of doing this, we just invoke the `orderbyBuilder` where the only change should be:
```
	const customOrderByKey =
		customSort?.[`${field}_${refScheme}`] ??
		customSort?.[field] ??
		(sortInfo.sortable === 'string' ? sortInfo.sortable : undefined);
```

Reply:
well we probably do need that, b/c as it is atm I don't any place where we add the asc/desc suffix